paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetTest.kt

```diff
@@ -126,6 +126,8 @@ internal class PaymentSheetTest {
         testContext.markTestSucceeded()
     }
 
+    // TODO: Add a test for
```

Comment:
👀

Reply:
f

Reply:
I wanted to do this, then figured it didn't make sense to do it in a whole integration test, so I added a new class for testing in isolation. But I forgot to remove the TODO.